Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -344,6 +344,34 @@ def search_incidents(client, args):
     return response
 
 
+def extract_data_form_other_fields_argument(other_fields: str, incident, changes):
```

Comment:
Let's be consistent, either add type annotations to all args or don't add at all and mention the type in the docstring